cli/create/templates/cartridge/{{ .Name }}-scm-1.rockspec

```diff
@@ -11,6 +11,7 @@ dependencies = {
     'cartridge == 2.4.0-1',
     'metrics == 0.6.1-1',
     'cartridge-cli-extensions == 1.1.0-1',
+    'prometheus == 1.0.4',
```

Comment:
Please, note that this module is deprecated (https://github.com/tarantool/prometheus#deprecated)

Use prometheus plugin from metrics.